exonum-java-binding/core/rust/src/runtime/config.rs

```diff
@@ -120,10 +123,10 @@ mod tests {
 
     #[test]
     fn not_forbidden_user_parameter() {
-        let validation_result = validate_and_convert("Duser.parameter=Djava.library.path");
+        let validation_result = validate_and_convert("Duser.parameter=Djava.class.path");
```

Comment:
Why is this test changed? We don't seem to allow `java.library.path` as a custom property.

Reply:
Thanks, this is the heritage of merging with the `ejb-app-packaging` branch that I just forgot to delete. Reverted.